### PR TITLE
Html

### PR DESCRIPTION
**Pull request by Aaron2278**
Html markup page based off of my previous readME markdown.

### CODE REVIEW

File: lesson_00/aaronsantiago/README.md
Comment (anthonydmays) [inline lesson_00/aaronsantiago/README.md:1]: This looks broken, please fix.

File: lesson_01/AaronSantiago/Aaron.html
Comment (anthonydmays) [inline lesson_01/AaronSantiago/Aaron.html:1]: There are some significant visual differences between your HTML version and your MARKDOWN version. The markdown version is borked for me, so I can't provide more detail than that ATM.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by Aaron2278 — COMMENTED** [review]
fixed the problem with the readME file and fixed something on the html file.

File: lesson_01/AaronSantiago/Aaron.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Personal README - Aaron Santiago</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css">
+  <link rel="stylesheet" href="Aaron.css">
+</head>
+<body>
+
+  <h1 class="animate__animated animate__fadeIn">Personal README - Aaron Santiago</h1>
```
Comment (anthonydmays) [inline lesson_01/AaronSantiago/Aaron.html:12]: There are a couple of diffs I'd like you to resolve. Please update the font and the title of this page (your name is in square brackets in the markdown version, for instance).

File: lesson_01/AaronSantiago/Aaron.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  <h2 class="animate__animated animate__fadeIn">Spare Time Rituals</h2>
+  <p class="animate__animated animate__fadeIn">In my precious moments outside of work, I cherish:</p>
+  <ul class="animate__animated animate__fadeIn">
+    <li><strong>Gaming Escapades:</strong> Whether it's exploring fantastical realms or teaming up for multiplayer adventures, gaming is my ultimate unwind.</li>
+    <li><strong>Culinary Ventures:</strong> The kitchen is my canvas. I enjoy experimenting with flavors and crafting delicious meals. Share your favorite recipes, and we'll swap culinary secrets!</li>
+    <li><strong>Friendship Conversations:</strong> Nothing beats a good conversation with friends. It could be about the latest gaming strategies, cooking tips, or just life in general. Feel free to join in!</li>
+  </ul>
+
+  <img class="animate__animated animate__fadeIn" src="https://remote-tools-images.s3.amazonaws.com/programmer-memes/11.jpg" alt="Programmer Meme">
```
Comment (anthonydmays) [inline lesson_01/AaronSantiago/Aaron.html:37]: This image is smaller than in the markdown version. Please adjust.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (anthonydmays) [inline .vscode/settings.json:1]: Please revert this change. You can use this article for help: https://www.freecodecamp.org/news/git-revert-file-reverting-a-file-to-a-previous-commit/

File: lesson_01/AaronSantiago/Aaron.html (hunk 2 above)
Comment (anthonydmays) [reply lesson_01/AaronSantiago/Aaron.html:37]: Still not quite there.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Comment by Aaron2278** [comment]
vscode folder, setting.json statement dupe should be resolved now.

File: .vscode/settings.json
Comment (anthonydmays) [reply .vscode/settings.json:1]: Still not addressed.

File: lesson_01/README.md
Comment (anthonydmays) [inline lesson_01/README.md:1]: Please revert this change.

File: lesson_01/AaronSantiago/Aaron.html (hunk 2 above)
Comment (anthonydmays) [reply lesson_01/AaronSantiago/Aaron.html:37]: Still not matching.

File: lesson_01/AaronSantiago/Aaron.html
Comment (anthonydmays) [reply lesson_01/AaronSantiago/Aaron.html:1]: This is unresolved, please continue making adjustments.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Pull request merged**